src/components/Dashboard.jsx

```diff
@@ -8,7 +8,11 @@ export const Dashboard = ({
 }) => {
   return (
     <>
+    <div className='mt-8 text-center mb-4 md:mb-0'>
```

Comment:
Adjust the margin-top/bottom for mobile as well. `mt-4 mb-2` on mobile should be fine.